python/tk_houdini_renderman/get_smart_frame_list.py

```diff
@@ -0,0 +1,105 @@
+# Source: https://github.com/BreakTools/Snippets/blob/main/get_smart_frame_list.py
```

Comment:
I'm not going to review this code, as its part of another repo. However, by adding the code to repo's manually, we will get code that is hard to maintain as the code is across multiple repo's. 

My guess is that this could be package that could be installed in the python packages (which could be defined in the requirements.txt). This makes it possible to use in any repo, while having the code in one place.